lib/datasource/git-submodules/index.ts

```diff
@@ -0,0 +1,55 @@
+import Git from 'simple-git/promise';
+import { URL } from 'url';
+
+import { ReleaseResult, PkgReleaseConfig, DigestConfig } from '../common';
+import { logger } from '../../logger';
+
+export async function getPkgReleases({
+  lookupName,
```

Comment:
```suggestion
  lookupName: string,
```

Reply:
Types come from `PkgReleaseConfig`. This is just destructing. Same on the line below.

Reply:
Ups, you are right. Sorry.